automatic/virtualbox_5.0.x/virtualbox_5.0.x.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>virtualbox</id>
```

Comment:
I believe it is yes, but this naming was suggested by @ferventcoder .
https://github.com/chocolatey/chocolatey-coreteampackages/issues/462#issuecomment-266245454
And since we don't create a new package on chocolatey, it really doesn't matter.